docs/_plugins/search_index.rb

```diff
@@ -263,7 +285,7 @@ def execute
       filename.gsub! /\A(\d{4})-(\d{2})-(\d{2})-(\w+)\.md\z/, '/\1/\2/\3/\4.html'
     end.compact.each do |id|
       puts "Removing #{id}..."
-      client.delete index: 'models', id: id, ignore: [404]
+      client.delete_by_query index: 'models', body: {query: {term: {url: id}}}
```

Comment:
I don't think we should change that.

Reply:
We need this, since we changed the _id in the indexed models